Resources/views/Navbar/navbar.html.twig

```diff
@@ -14,7 +14,7 @@
         </div>
         {% endblock header %}
         {% block menu_container %}
-        <div class="collapse navbar-collapse navbar-responsive-collapse">
+        <div class="collapse navbar-collapse navbar-responsive-collapse-{% if id|default(false) %}{{ id }}{% endif %}">
```

Comment:
Should be:

`navbar-responsive-collapse{% if id|default(false) %}-{{ id }}{% endif %}`

For both of them. Don't put the hyphen at the end unless there's an ID